sandbox/registration-processor-camel-routes-new-mz.xml

```diff
@@ -191,11 +191,11 @@
             message="demo-dedupe-->bio-dedupe new route ${bodyAs(String)}" />
         <choice>
             <when>
-                <simple>${bodyAs(String)} contains '"isValid":true'</simple>
+		<jsonpath suppressExceptions="true">$.[?(@['tags']['ID_OBJECT-presentCity'] != 'Tubo' &amp;&amp; @['isValid'] == true)]</jsonpath> 
                 <to uri="eventbus:bio-dedupe-bus-in" />
             </when>
             <when>
-                <simple>${bodyAs(String)} contains '"address":"abis-handler-bus-in"'</simple>
+		<jsonpath suppressExceptions="true">$.[?(@['tags']['ID_OBJECT-presentCity'] != 'Tubo' &amp;&amp; @['address'] == 'abis-handler-bus-in')]</jsonpath>
                 <to uri="eventbus:abis-handler-bus-in" />
             </when>
 			<when>
```

Comment:
I see the condition for manual verification is not done, we  don't have packets to stop before manual verification stage ? If yes use a different city here